src/App.css

```diff
@@ -1,28 +1,180 @@
-.app {
-  height: 100vh;
-  width: 100vw;
-  transition: background-color 1s ease-in;
+/*  Search bar */
+
+body {
+  margin: 0;
+}
+.flex-container {
+  background-color: #8ccef4;
+  width: 1fr;
+  height: 1fr;
+  display: -ms-flexbox;
+
+  display: flex;
+
+  flex-direction: column;
+
+  flex-wrap: nowrap;
+
+  justify-content: stretch;
+
+  align-content: flex-start;
+
+  align-items: stretch;
+}
+
+.flex-item:nth-child(1) {
+  order: 0;
+
+  flex: 0 1 auto;
+  -webkit-align-self: auto;
+  -ms-flex-item-align: auto;
+  align-self: auto;
+}
+
+.flex-item:nth-child(2) {
+  order: 0;
+
+  flex: 0 1 auto;
+
+  align-self: auto;
+}
+
+.flex-item:nth-child(3) {
+  order: 0;
+
+  flex: 0 1 auto;
+
+  align-self: auto;
+}
+
+.flex-item:nth-child(4) {
```

Comment:
Remember we don't want any CSS classes repeating the same CSS rules. If they do the same thing, you only need one.